src/stages/stage3_events.c

```diff
@@ -250,7 +250,7 @@ static int stage3_burstfairy(Enemy *e, int t) {
 
 	charge_effect(e, t, bursttime);
 
-	FROM_TO_SND("shot1_loop", bursttime, bursttime + burstspan, step) {
+	FROM_TO_SND("shot1_loop", bursttime, bursttime + burstspan * step - 1, step) {
```

Comment:
Not sure about this one. Seems fine during testing???

Reply:
What is this change trying to solve?

Reply:
```
../src/stages/stage3_events.c: In function ‘stage3_burstfairy’:
../src/stage.h:35:101: error: assuming signed overflow does not occur when assuming that (X + c) >= X is always true [-Werror=strict-overflow]
 #define FROM_TO(start,end,step) _ni = _ni; _i = (*__timep - (start))/(step); if(*__timep >= (start) && *__timep <= (end) && !((*__timep - (start)) % (step)))
                                                                                 ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
../src/stage.h:58:35: note: in expansion of macro ‘FROM_TO’
 #define PLAY_FOR(name,start, end) FROM_TO(start,end,2) { play_sfx_loop(name); }
                                   ^~~~~~~
../src/stage.h:61:41: note: in expansion of macro ‘PLAY_FOR’
 #define FROM_TO_SND(snd,start,end,step) PLAY_FOR(snd,start,end); FROM_TO(start,end,step)
                                         ^~~~~~~~
../src/stages/stage3_events.c:253:2: note: in expansion of macro ‘FROM_TO_SND’
  FROM_TO_SND("shot1_loop", bursttime, bursttime + burstspan, step) {
  ^~~~~~~~~~~
```

Reply:
I think we should just disable this warning globally. It's mostly useless and has lots of false positives. 

The changed line needs to be reverted, since it alters behavior.

Reply:
Disabled it globally with `-Wstrict-overflow=0`.